package.json

```diff
@@ -7,6 +7,9 @@
 		"npm": ">=8.19.0"
 	},
 	"dependencies": {
+		"@emotion/react": "^11.13.3",
+		"@emotion/styled": "^11.13.0",
+		"@mui/material": "^6.0.2",
```

Comment:
<!-- Highlight something to the reader.  -->
**note (non-blocking):** Normally, I'd say to be sure to remove packages you decide not to use with `npm uninstall`, but I know we're using the same package in the other branch so it's all good 👍